src/util/log.c

```diff
@@ -15,6 +16,56 @@
 #define unlikely(x)     (x)
 #endif
 
+/**
+ * Compares two strings byte by byte and ignores the
+ * character's case. Stops at the n-th byte of both
+ * strings.
+ * 
+ * This is basically a replacement of the POSIX-function
+ * _strncasecmp_. Since tpm2-tss is supposed to be compatible
+ * with ISO C99 and not with POSIX, _strncasecmp_ had to be
+ * replaced. This function creates lowercase representations
+ * of the strings to compare and calls the function _strncmp_
+ * on them.
+ *
+ * @param string1 The first of the two strings to compare
+ * @param string2 The second of the two strings to compare
+ * @param n The maximum number of bytes to compare
+ * @return 0 if both strings are equal (case insensitive),
+ *  an integer greater than zero if string1 is greater than
+ *  string 2 and an integer smaller than zero if string1 is
+ *  smaller than string2
+ *
+ */
+static int caseInsensitiveStrncmp(const char *string1, 
```

Comment:
Could you rename it to something that doesn't use camel case please.
See the recommendation in https://github.com/tpm2-software/tpm2-tss/blob/master/doc/coding_standard_c.md#naming-variables-functions-and-other-stuff

Reply:
Augh....yes, I should have seen that right away. Sorry. The latest commit should contain the necessary changes.